### ISSUE TITLE
isInverted is false in all azure-themes

### ISSUE THREAD

**Issue by DavidParks8** [microsoft/fluentui#16376]
The [AzureThemeDark](https://github.com/microsoft/fluentui/blob/d0d59e633912d2fe72a48e1472afe220a74a51d0/packages/azure-themes/src/azure/AzureThemeDark.ts#L129) does not set `isInverted` and thus does not accurately reflect the theme "inversion" state. The same issue is present in the high contrast dark theme. 

There are a number of issues with these azure themes that we've identified and would like to fix at the source, but the reality is that we need to enable azure portal extensions to be able to fix themselves faster than the fixes can be identified and merged into this repo. Setting `isInverted` properly is the first step to enabling dark theme fixes that don't incur too much technical debt in extensions and unblocks them to fix the issues they find on their own while they wait for the official fixes to be merged to this repo. Without `isInverted` being set correctly, we fear the fixes extensions attempt will be too brittle/buggy and thus have deemed it a "blocker" for releasing many react-based blades in portal.azure.com.

#### Actual behavior:

AzureThemeDark.isInverted === false
AzureThemeHighContrastDark.isInverted === false

#### Expected behavior:

AzureThemeDark.isInverted === true
AzureThemeHighContrastDark.isInverted === true

### Priorities and help requested:

Are you willing to submit a PR to fix? (Yes)

Requested priority: (Blocking)

Products/sites affected: (portal.azure.com)

**Comment by layershifter** [comment]
Was fixed #16378 🎉
